gateway/ln-gateway/src/lightning/lnd.rs

```diff
@@ -787,12 +787,25 @@ impl ILnRpcClient for GatewayLndClient {
                             0
                         };
 
+                    let (channel_point_txid, channel_point_output_index) = channel
+                        .channel_point
+                        .split_once(':')
+                        .map(|(txid, output_index)| {
+                            (
+                                txid.to_string(),
+                                output_index.parse::<u32>().unwrap_or_default(),
+                            )
+                        })
+                        .unwrap_or_default();
```

Comment:
We could actually save that custom parsing logic by just using [`OutPoit::from_str`](https://docs.rs/bitcoin/latest/bitcoin/struct.OutPoint.html#impl-FromStr-for-OutPoint)